datasafe-simple-adapter/datasafe-simple-adapter-impl/src/main/java/de/adorsys/datasafe/simple/adapter/impl/SimpleDatasafeServiceImpl.java

```diff
@@ -117,7 +117,8 @@ public SimpleDatasafeServiceImpl(DFSCredentials dfsCredentials) {
                     )
                     .enablePathStyleAccess();
 
-            boolean useEndpoint = (!amazonS3DFSCredentials.getUrl().equals(AMAZON_URL));
+            boolean useEndpoint = !amazonS3DFSCredentials.getUrl().equals(AMAZON_URL)
```

Comment:
Script should print that it allows AMAZON_URL too, not only amazon S3, but also i.e. CEPH could be tested with it

Reply:
I've added this to script help